src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -112,6 +112,12 @@ private readonly static Lazy<PreferenceSettings>
         /// </summary>
         public static readonly DateTime DynamoDefaultTime = new DateTime(1977, 4, 12, 12, 12, 0, 0);
 
+        internal static readonly IEnumerable<string> InitialExperimentalLib_Namespaces =
+        [
+            //TODO remove tsplines - it's no longer experimental.
+            "ProtoGeometry.dll:Autodesk.DesignScript.Geometry.TSpline",
```

Comment:
Thanks, I thought I removed the TSpline feature flag check would be enough, I guess not..